engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/Function.scala

```diff
@@ -45,14 +45,25 @@ object Function {
     */
   sealed case class Lambda(
     override val arguments: List[DefinitionArgument],
-    override val body: Expression,
+    val bodyList: Seq[Expression],
```

Comment:
```suggestion
    val bodySeq: Seq[Expression],
```
:)

Reply:
Renamed to `bodySeq` in [13ae27e](https://github.com/enso-org/enso/pull/8207/commits/13ae27e8a86d4b645b5b0be3ecc220f068de1322)